packages/financial-templates-lib/test/price-feed/CreatePriceFeed.js

```diff
@@ -1221,4 +1223,38 @@ contract("CreatePriceFeed.js", function(accounts) {
       null
     );
   });
+  it("Valid DefiPulse config", async function() {
+    const config = {
+      type: "defipulse",
+      lookback: 604800,
+      apiKey: apiKey,
+      minTimeBetweenUpdates: 600,
+      project: "SushiSwap"
+    };
+
+    const validDefiPulsePriceFeed = await createPriceFeed(logger, web3, networker, getTime, config);
+
+    assert.isTrue(validDefiPulsePriceFeed instanceof DefiPulsePriceFeed);
+  });
+
+  it("Invalid DefiPulse config", async function() {
+    const validConfig = {
+      type: "defipulse",
+      lookback: 604800,
+      apiKey: apiKey,
+      minTimeBetweenUpdates: 600,
+      project: "SushiSwap"
+    };
+
+    assert.equal(await createPriceFeed(logger, web3, networker, getTime, { ...validConfig, apiKey: undefined }), null);
+    assert.equal(await createPriceFeed(logger, web3, networker, getTime, { ...validConfig, project: undefined }), null);
```

Comment:
Related to the comment I had about sanity checking `project`, can you add some tests for invalid project names like "Sushi"

Reply:
Done